lib/wsdl.js

```diff
@@ -2081,14 +2082,20 @@ WSDL.prototype._fromServices = function(services) {
 
 WSDL.prototype._splitQName = splitQName;
 
-WSDL.prototype._xmlnsMap = function() {
+WSDL.prototype._xmlnsMap = function (xmlnsIntoEnvelope) {
   var xmlns = this.definitions.xmlns;
   var str = '';
   for (var alias in xmlns) {
     if (alias === '' || alias === TNS_PREFIX) {
       continue;
     }
+
     var ns = xmlns[alias];
+    if (ns && xmlnsIntoEnvelope && ~xmlnsIntoEnvelope.indexOf(alias)) {
```

Comment:
I just wrote a little "proof of concept" that `_.includes()` won't fail if `xmlnsIntoEnvelope` is not given over at the lodash docs page:

```javascript
function test (xmlnsIntoEnvelope) {
  return _.includes(xmlnsIntoEnvelope, 'testNamespace');
}

// call it without params
test(); // the output is "false",

// with a string that contains the given namespace
test('mytestNamespace');  // the output is "true"
```

so, I think we are save to go with the lodash function 😃